scm-webapp/src/main/resources/sonia/scm/cli/i18n_de.properties

```
@@ -79,3 +84,102 @@ repoDeletePrompt= Wenn dieses Repository endg
 ### Modify repo
 scm.repo.modify.usage.description.0 = Aktualisiert ein Repository auf dem Server
 scm.repo.modify.repository = Repository Namespace/Name
+
+## User
+scm.user.username = Benutzername
+scm.user.displayName = Anzeigename
+scm.user.email = E-Mail-Adresse
+scm.user.external = Extern
+scm.user.password = Passwort
+scm.user.active = Aktiv
+
+### User error
+scmUserErrorPassword = F�r interne Benutzer muss ein Passwort gesetzt werden
+scmUserErrorExternalActivate = Externe Benutzer k�nnen nicht aktiviert werden
+scmUserErrorExternalDeactivate = Externe Benutzer k�nnen nicht deaktiviert werden
+scmUserErrorNotFound= Benutzer konnte nicht gefunden werden
+
+### User command
+scm.user.usage.description.0 = Gruppen Befehl f�r alle User Befehle
+
+### Get user
+scm.user.get.usage.description.0 = Liefert Informationen zum Benutzer
+
+### List user
+scm.user.list.usage.description.0 = Listet alle Benutzer auf dem Server
+
+### Create user
+scm.user.create.usage.description.0 = Erzeugt einen neuen Benutzer auf dem Server
+scm.user.create.deactivate = Benutzer ist deaktiviert
+scm.user.create.external = Der Benutzer wird von einem Fremdsystem verwaltet
+
+### Delete user
+scm.user.delete.usage.description.0 = L�scht einen Benutzer auf dem Server
+scm.user.delete.prompt = Setzen Sie diese Option, um einen Benutzer endg�ltig zu l�schen
+scmUserDeleteConfirm = Wenn dieser Benutzer endg�ltig gel�scht werden soll, setzen Sie bitte --yes
+scmUserDeleteSuccess = Benutzer erfolgreich gel�scht
+
+### Modify user
+scm.user.modify.usage.description.0 = Aktualisiert einen Benutzer auf dem Server
+
+### Activate user
+scm.user.activate.usage.description.0 = Aktiviert einen Benutzer auf dem Server
+
+### Deactivate user
+scm.user.deactivate.usage.description.0 = Deaktiviert einen Benutzer auf dem Server
+
+### Convert to external user
+scm.user.convert-to-external.usage.description.0 = Konvertiert einen Benutzer zu einem externen Benutzer
+
+### Convert to internal user
+scm.user.convert-to-internal.usage.description.0 = Konvertiert einen Benutzer zu einem internen Benutzer
+
+### Group
+scm.group.name = Name
+scm.group.description = Beschreibung
+scm.group.members = Mitglieder
+scm.group.membersList = Mitglieder
+scm.group.external = Extern
+
+### Group command
+scm.group.usage.description.0 = Hauptbefehl f�r alle Gruppen Befehle
```

```suggestion
scm.group.usage.description.0 = Gruppenbefehl für alle \"Gruppen\" Befehle
```

Maybe like this to make it more consistent?

---

I don't like neither `Gruppenbefehl` nor "Hauptbefehl". Even "Parent command" sounds a bit weird, but we couldn't find a better way to express what the command does.